### ISSUE TITLE
Support for gamepad API

### ISSUE THREAD

**Issue by alexjstubbs** [resin-io-projects/resin-wpe#17]
could this be built to support the HTML5 Gamepad API?

**Comment by petrosagg** [comment]
@alexjstubbs From what I see in [the recipe](https://github.com/WebPlatformForEmbedded/meta-wpe/blob/5920963af33db8ca32f3c16d6b10375a742ae468/recipes-wpe/wpewebkit/wpewebkit_2.20.bb#L37) used for the base-image the Gamepad API is enabled by default. Unfortunately I don't have a gamepad to test.

I just updated WPEWebkit to the latest version, do you want to try it in the latest image?

**Comment by alexjstubbs** [comment]
I will try with the latest this evening but I did also see that it was enabled by default but http://html5test.com reported that it is unsupported and online gamepad testers do not initialize or detect any gamepad. I got this working with qtbrowser once but that was with buildroot, which I cannot get to work with WPE. Not so hot with Yocto yet.

Anyway, I will see if anything has changed with the latest image.
